### PR TITLE
dependency refactoring

### PR DESCRIPTION
**Pull request by elyutikov**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    val doobieCore   = "org.tpolecat" %% "doobie-core"     % Versions.doobie
+    val doobiePg     = "org.tpolecat" %% "doobie-postgres" % Versions.doobie
+    val doobieHikari = "org.tpolecat" %% "doobie-postgres" % Versions.doobie
+
+    val circeCore    = circe("circe-core")
+    val circeGeneric = circe("circe-generic")
+    val circeParser  = circe("circe-parser")
+    val circeRefined = circe("circe-refined")
+
+    val cirisCore    = ciris("ciris")
```
Comment (Ssstlis) [inline project/Dependencies.scala]: Why isn't a pureconfig?

File: project/Dependencies.scala [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    val http4sClient = http4s("http4s-blaze-client")
+    val http4sCirce  = http4s("http4s-circe")
+
+    val http4sJwtAuth = "dev.profunktor" %% "http4s-jwt-auth" % Versions.http4sJwtAuth
+
+    val refinedCore = "eu.timepit" %% "refined"      % Versions.refined
+    val refinedCats = "eu.timepit" %% "refined-cats" % Versions.refined
+
+    val log4cats = "io.chrisdavenport" %% "log4cats-slf4j" % Versions.log4cats
+    val newtype  = "io.estatico"       %% "newtype"        % Versions.newtype
```
Comment (Ssstlis) [inline project/Dependencies.scala]: 😎

File: project/Dependencies.scala [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+    // Compiler plugins
```
Comment (Ssstlis) [inline project/Dependencies.scala]: I think we should separate path-deps from compiler plugins and test deps

File: project/Dependencies.scala [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    val betterMonadicFor = "com.olegpy"    %% "better-monadic-for" % Versions.betterMonadicFor
+    val kindProjector    = "org.typelevel" % "kind-projector"      % Versions.kindProjector
+
+    // Runtime
+    val logback = "ch.qos.logback" % "logback-classic" % Versions.logback
```
Comment (Ssstlis) [inline project/Dependencies.scala]: What do you think about log4cats? Or we just start to use tofu-logging?

File: .scalafmt.conf [hunk 5]
```diff
@@ -0,0 +1,22 @@
+version                                      = "2.4.2"
```
Comment (Ssstlis) [inline .scalafmt.conf]: Can we configure to keep up with curvy braces around single expression?
Also i sew down that this configuration create a single space in curvy braces import. Don't look so good =)

Also with this config we should to create aliases in build.sbt to fast checking and reformat code from tasks.

Comment (Ssstlis) [reply .scalafmt.conf]:
```
addCommandAlias("fmt", "all scalafmtSbt scalafmt test:scalafmt")
addCommandAlias("fmtCheck", "all scalafmtSbtCheck scalafmtCheck test:scalafmtCheck")
```

File: project/Dependencies.scala (hunk 1 above)
Comment (elyutikov) [reply project/Dependencies.scala]: I was based on this [article](https://typelevel.org/blog/2017/06/21/ciris.html). In fact, I also want to try something new)

File: project/Dependencies.scala (hunk 4 above)
Comment (elyutikov) [reply project/Dependencies.scala]: > What do you think about log4cats? Or we just start to use tofu-logging?

I agree to tofu-logging

File: project/Dependencies.scala (hunk 1 above)
Comment (Ssstlis) [reply project/Dependencies.scala]: Ok, looks interesting. If it could to F-loading, it will be very good)

File: project/Dependencies.scala (hunk 3 above)
Comment (elyutikov) [reply project/Dependencies.scala]: > I think we should separate path-deps from compiler plugins and test deps

Ok, I'll divide them into different objects.

**Pull request merged**